fluentui_drawer/src/main/java/com/microsoft/fluentui/persistentbottomsheet/PersistentBottomSheet.kt

```diff
@@ -257,19 +233,21 @@ class PersistentBottomSheet @JvmOverloads constructor(context: Context, attrs: A
 
     fun expand(focusDrawerHandle: Boolean = true) {
         persistentSheetBehavior.state = BottomSheetBehavior.STATE_EXPANDED
+        setClickListenerForDismissal()
```

Comment:
Cant we have this somewhere in init block?
That way. we do not need to set it individually for expand/show
Also, i feel when the sheet is gone then clicklistener wont work. so, this would work for us and we will not have to call it everytime the sheet is expanded.

Reply:
Sheet outline is there even after collapse but it's transparent and so when items are collapsed then we want to remove this listener which is done by setting clickable/focusable as false.